_oasis

```diff
@@ -29,8 +29,53 @@ Library xenopsd
   Findlibname:        xenopsd
   ByteOpt:            -warn-error +a
   NativeOpt:          -warn-error +a
-  Modules:            Suspend_image, Cancellable_subprocess, Bootloader, Ionice, Mac, Xenops_migrate, Xenops_hooks, Task_server, Xenops_task, Updates, Xenops_utils, Xenops_server, Xenopsd, Xenops_server_plugin, Xenops_server_skeleton, Scheduler, Path, Storage, Interface, Xenctrl_uuid, Xenstore, Version
-  BuildDepends:       threads, threads.posix, uuidm, xmlm, cohttp, uri, rpclib, rpclib.syntax, forkexec, fd-send-recv, xcp, xcp.xen, xcp.storage, sexplib, sexplib.syntax, xcp, uutf, xenstore, xenstore.unix, xenstore_transport, xenstore_transport.unix, oclock
+  Modules:            
+    Suspend_image,
+    Cancellable_subprocess,
+    Bootloader,
+    Ionice,
+    Mac,
+    Xenops_migrate,
+    Xenops_hooks,
+    Task_server,
+    Xenops_task,
+    Updates,
+    Xenops_utils,
+    Xenops_server,
+    Xenopsd,
+    Xenops_server_plugin,
+    Xenops_server_skeleton,
+    Scheduler,
+    Path,
+    Storage,
+    Interface,
+    Xenctrl_uuid,
+    Xenstore,
+    Version
+  BuildDepends:
+    bisect_ppx,       
+    threads,
+    threads.posix,
+    uuidm,
+    xmlm,
+    cohttp,
+    uri,
+    rpclib,
+    rpclib.syntax,
+    forkexec,
+    fd-send-recv,
+    xcp,
+    xcp.xen,
+    xcp.storage,
+    sexplib,
+    sexplib.syntax,
+    xcp,
+    uutf,
+    xenstore,
+    xenstore.unix,
+    xenstore_transport,
+    xenstore_transport.unix,
+    oclock
```

Comment:
I'm glad you have split these out onto separate lines!   Much better for resolving merges. :)